geniza/corpus/templates/corpus/document_scholarship.html

```diff
@@ -20,33 +20,43 @@ <h1 class="sr-only">{{ page_title }}</h1>
-                                    {% blocktranslate with location=fn.location url=fn.url trimmed %}
-                                        see <a href="{{ url }}">{{ location }}</a> for
-                                    {% endblocktranslate %}
-                                {% elif fn.location %}
-                                    {# Translators: label for footnote location within source #}
-                                    {% blocktranslate with location=fn.location trimmed %}
-                                        see {{ location }} for
-                                    {% endblocktranslate %}
-                                {% elif fn.url %}
-                                    {# Translators: label for included document relations for a footnote #}
-                                    <a href="{{ fn.url }}">{% translate 'includes' %}</a>
-                                {% else %}
-                                    {% translate 'includes' %}
-                                {% endif %}
-                            </dt>
-                            <dd class="relation">
-                                {{ fn.doc_relation }}
-                            </dd>
+                            {% with document|footnotes_on_source:source as footnotes %}
+                                <dt class="relation">
+                                    {% if footnotes|length == 1 %}
+                                        {% with fn=footnotes.0 single="true" %}
+                                            {% if fn.location or fn.url %}
+                                                {% include "corpus/snippets/footnote_location.html" %}
+                                            {% else %}
+                                                {# Translators: label for included document relations for a footnote #}
+                                                {% translate "includes" %}
+                                            {% endif %}
+                                        {% endwith %}
+                                    {% else %}
+                                        see
+                                        <ul>
+                                            {% for fn in footnotes %}
+                                                <li class="location">
+                                                    {% include "corpus/snippets/footnote_location.html" %}
+                                                </li>
+                                            {% endfor %}
+                                        </ul>
+                                        for
+                                    {% endif %}
+                                </dt>
+                                <dd class="relation">
+                                    {% if footnotes|length == 1 %}
+                                        {{ footnotes.0.doc_relation }}
```

Comment:
for now let's just assume we can rely on the document relationship from the first footnote — they way our data is now, and certainly for the specific problem we're trying to solve, the document relationships will all be the same for footnotes from the same source

Reply:
Done in e0b997a7e7c2ef999a88c6c00b2194d0d9e69071 👍